OpenRA.Mods.Common/Widgets/Logic/MusicPlayerLogic.cs

```diff
@@ -41,12 +41,12 @@ public MusicPlayerLogic(Widget widget, Ruleset modRules, World world, Action onE
 			var playButton = panel.Get<ButtonWidget>("BUTTON_PLAY");
 			playButton.OnClick = Play;
 			playButton.IsDisabled = noMusic;
-			playButton.IsVisible = () => !Sound.MusicPlaying;
+			playButton.IsVisible = () => { !Sound.MusicPlaying && currentSong != null; };
```

Comment:
Wouldn't implementing #8746 as part of this PR fell into scope-creeping or still acceptable?